cybershell/adaptive/hitl.py

```
@@ -0,0 +1,26 @@
+import numpy as np
+from .mapper import AdaptiveLearningMapper, FAMILIES
+from .signals import SignalEvent
+@dataclass
+class FeedbackItem:
+    event: SignalEvent
+    correct_families: List[str]
+@dataclass
+class HITLLoop:
+    mapper: AdaptiveLearningMapper
+    buffer: List[FeedbackItem] = field(default_factory=list)
+    def submit_feedback(self, item: FeedbackItem):
```

Ruff (0.12.2)

Missing return type annotation for public function `submit_feedback`

Add return type annotation: `None`

(ANN201)